pkgs/development/python-modules/Rtree/default.nix

```
@@ -12,13 +12,12 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ libspatialindex ];
 
   patchPhase = ''
-    substituteInPlace rtree/core.py --replace \
+    substituteInPlace rtree/finder.py --replace \
       "find_library('spatialindex_c')" "'${libspatialindex}/lib/libspatialindex_c${stdenv.hostPlatform.extensions.sharedLibrary}'"
   '';
 
-  # Tests appear to be broken due to mysterious memory unsafe issues. See #36760
-  doCheck = false;
-  checkInputs = [ numpy ];
+  checkInputs = [ numpy pytestCheckHook ];
```

Same reasoning as above.

```suggestion
  checkInputs = [
    numpy
    pytestCheckHook
  ];
```